File: 20.cpp
//Factorial digit sum
#include<iostream>

using namespace std;
int main(){
    int n=100;
    double pro=1;
    int sum=0;
    double digit=0;
    for(int i=n;i>=1;i--){
        pro*=i;
    }
    while(pro>0){
        digit=((int)pro)%10;
        cout<<"digit"<<digit<<endl;
        sum+=digit;
        pro=((int)pro)/10;
    }
    //cout<<pro;
}

File: 20.js
let n=100;let pro=1
for(let i=n;i>=1;i--){
    pro*=i;
}
let digit=0;
let sum=0;
while(pro>0){
    digit=pro%10;
    
    sum+=digit;
    pro=pro/10;
}
console.log(sum);



